src/images/neo4j.rs

```diff
@@ -74,8 +77,70 @@ impl Neo4jImage {
             &graph_data_science_bytes,
         )
         .await?;
+
         Ok(())
     }
+    pub async fn post_startup(&self, _proj: &str, docker: &Docker) -> Result<()> {
```

Comment:
I think I might ditch this post startup stuff and just manually migrate since it seems to be quite a challenge, since I need neo4j to off to migrate and if I stop neo4j the container goes down